packages/main/test/samples/Toast.sample.html

```diff
@@ -0,0 +1,190 @@
+<header class="component-heading">
+	<div class="control-header">Toast</div>
+	<div class="component-heading-since">
+		<span><!--since_tag_marker--></span>
+	</div>
+</header>
+<div class="control-tag">&lt;ui5-toast&gt;</div>
```

Comment:
Add the package info after the `<header` tag and before the `<div class="control-tag">`.
```html
<div class="component-package">@ui5/webcomponents</div>
```

We started adding the info just recently.